nservicebus/recoverability/configure-error-handling.md

```diff
@@ -52,7 +53,7 @@ partial:errorheader
 
 Administrators should monitor the error queue, in order to detect when problems occur. The message in the error queue contains information regarding its source queue and machine. Having that information an administrator can investigate the specific node and solve the problem, e.g. bring up a database that went down.
 
-Monitoring and handling of failed messages with [ServicePulse](/servicepulse/) provides access to full exception details (including stack-trace). [ServiceInsight](/serviceinsight/) additionally enables advanced debugging with a full message context. Both of them provide a `retry` functionality to send the message back to the endpoint for re-processing. For more details, see [Introduction to Failed Messages Monitoring in ServicePulse](/servicepulse/intro-failed-messages.md).
+Monitoring and handling of failed messages with [ServicePulse](/servicepulse/) provides access to full exception details (including stack-trace). [ServiceInsight](/serviceinsight/) enables additional, advanced debugging with providing full message processing context. Both of them provide a `retry` functionality to send the message back to the endpoint for re-processing. For more details, see [Introduction to Failed Messages Monitoring in ServicePulse](/servicepulse/intro-failed-messages.md).
```

Comment:
Not really happy with "additional" and also what does "full message processing context" even mean?

Should we rewrite this to something like:

In addition to providing information about exception details [ServiceInsight](/serviceinsight/) offers advanced debugging capability given insight into the full message flow from where the message originally started to where the message was handled.

Or something like that?